pytorch_toolkit/human_pose_estimation/README.md

```diff
@@ -1,59 +1,80 @@
 # Real-time 2D Multi-Person Pose Estimation on CPU: Lightweight OpenPose
 
-This repository contains training code for the paper [Real-time 2D Multi-Person Pose Estimation on CPU: Lightweight OpenPose](https://arxiv.org/pdf/1811.12004.pdf). This work heavily optimizes the [OpenPose](https://github.com/CMU-Perceptual-Computing-Lab/openpose) approach to reach real-time inference on CPU with negliable accuracy drop. It detects a skeleton (which consists of keypoints and connections between them) to identify human poses for every person inside the image. The pose may contain up to 18 keypoints: ears, eyes, nose, neck, shoulders, elbows, wrists, hips, knees, and ankles. On COCO 2017 Keypoint Detection validation set this code achives 40% AP for the single scale inference (no flip or any post-processing done). The result can be reproduced using this repository.
+This repository contains training code for the paper [Real-time 2D Multi-Person Pose Estimation on CPU: Lightweight OpenPose](https://arxiv.org/pdf/1811.12004.pdf). This work heavily optimizes the [OpenPose](https://github.com/CMU-Perceptual-Computing-Lab/openpose) approach to reach real-time inference on CPU with a negligible accuracy drop. It detects a skeleton (which consists of keypoints and connections between them) to identify human poses for every person inside the image. The pose may contain up to 18 keypoints: ears, eyes, nose, neck, shoulders, elbows, wrists, hips, knees, and ankles. This code achieves a 40% AP for the single-scale inference on the COCO 2017 Keypoint Detection validation set without a flip or post-processing.You can reproduce the result using this repository.
 
-The implementation of our paper for single-person pose estimation problem [Global Context for Convolutional Pose Machines](https://arxiv.org/pdf/1906.04104.pdf) is added. See [README_single.md](README_single.md) for the details.
+The implementation of our paper for single-person pose estimation problem [Global Context for Convolutional Pose Machines](https://arxiv.org/pdf/1906.04104.pdf) is added. See [README_single.md](README_single.md) for details.
 
-<p align="center">
-  <img src="data/preview.jpg" />
-</p>
+![](data/preview.jpg)
 
 ## Table of Contents
 
 * [Requirements](#requirements)
 * [Prerequisites](#prerequisites)
 * [Training](#training)
 * [Validation](#validation)
-* [Pre-trained model](#pre-trained-model)
-* [Output format](#output-format)
-* [C++ demo](#cpp-demo)
-* [Python demo](#python-demo)
-* [Fine-tuning](#fine-tuning)
+* [Pretrained Model](#pre-trained-model)
+* [Output Format](#output-format)
+* [C++ Demo](#cpp-demo)
+* [Python Demo](#python-demo)
+* [Fine-Tuning](#fine-tuning)
```

Comment:
I believe `tuning` needs to be in lower case, since fine-tuning is 1 word.

Reply:
No, in this case we do have to capitalize the word after the hyphen according to our [style guide ](https://software.intel.com/en-us/dx-hub/copy-style-guide#headline_style_capitalization)

Reply:
I cannot open this link. Ok if you are sure.